src/Yandex.Alice.Sdk/Models/AliceEntityGeoValueModel.cs

```diff
@@ -6,6 +6,12 @@
     [UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
     public class AliceEntityGeoValueModel
     {
+        [JsonPropertyName("country")]
+        public string Country { get; set; }
+
+        [JsonPropertyName("city")]
+        public string City { get; set; }
+
```

Comment:
Thank you for your contribution! :)
Does it make sense to add an `airport` field too?